src/gui/search/searchsortmodel.h

```diff
@@ -49,8 +52,45 @@ class SearchSortModel: public QSortFilterProxyModel
 
     explicit SearchSortModel(QObject *parent = 0);
 
+    void enableNameFilter(bool enabled);
+    void setNameFilter(const QString& searchTerm = QString());
+
+    //! \brief setSizeFilter Sets parameters for filtering by size
```

Comment:
These are [Doxygen](http://www.doxygen.org/) comments. Since you work on Windows you might know C# XML comments which appear in generated documentation and in the IDE tooltips. These serve the same purpose: Doxygen can generate documentation and good IDE shows these comments. For instance: 
![kdevelop](https://cloud.githubusercontent.com/assets/394621/14178868/dea05888-f75c-11e5-8114-cba21e8ee74e.png)